app/config/AmlsGlobal.scala

```diff
@@ -22,18 +22,36 @@ import play.api.{Application, Configuration, Play}
 import uk.gov.hmrc.play.config.RunMode
 import uk.gov.hmrc.play.microservice.bootstrap.DefaultMicroserviceGlobal
 
+//TODO: Remove this file when upgrading to Play 2.6 - see https://www.playframework.com/documentation/2.7.x/GlobalSettings
```

Comment:
Why can we not remove it quite yet?

Reply:
It's the entry point for the application, extending from a class in a deprecated version of HMRC's [microservice bootstrap](https://github.com/hmrc/frontend-bootstrap). To [remove the object and move fully to DI](https://www.playframework.com/documentation/2.7.x/GlobalSettings), we'll need to perform the upgrade to Play 2.6 via a [newer version of bootstrap](https://github.com/hmrc/bootstrap-play-26). That work is covered by other tickets (e.g. [here](https://jira.tools.tax.service.gov.uk/browse/AMLS-4960)).